Sitemap/DepthGuesser.php

```diff
@@ -0,0 +1,66 @@
+<?php
+
+/*
+ * This file is part of the Symfony CMF package.
+ *
+ * (c) 2011-2016 Symfony CMF
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Cmf\Bundle\SeoBundle\Sitemap;
+
+use Doctrine\Bundle\PHPCRBundle\ManagerRegistry;
+use Doctrine\ODM\PHPCR\DocumentManager;
+use Symfony\Cmf\Bundle\SeoBundle\Model\UrlInformation;
+
+/**
+ * This guesser will add the depth of a document persisted on a phpcr node.
+ *
+ * @author Maximilian Berghoff <Maximilian.Berghoff@mayflower.de>
+ */
+class DepthGuesser implements GuesserInterface
+{
+    /**
+     * @var ManagerRegistry
+     */
+    private $managerRegistry;
+
+    /**
+     * @var int The depth of the content base path as the depth offset.
+     */
+    private $offset;
+
+    /**
+     * DepthGuesser constructor.
+     *
+     * @param ManagerRegistry $managerRegistry
+     */
+    public function __construct(ManagerRegistry $managerRegistry, $contentBasePath)
+    {
+        $this->managerRegistry = $managerRegistry;
+        $this->offset = substr_count($contentBasePath, '/') + 1;
```

Comment:
contentBasePath: /cms/content => offset = 2 +1, contentDocumentPath: /cms/content/test-document => depth = 3 ==> 3 - 3 = 0 => the root on the contentpath.

contentBasePath: /cms/content => offset = 2 +1, contentDocumentPath: /cms/content/collection/test-document => depth = 4 ==> 4 - 3 = 1 => the content will be dispplayed with an indent of 1

contentBasePath: / => offset = 1 +1 = 2, contentDocumentPath: /test-document => depth = 1 ==> 1 - 2 = 0 => @dbu you are right :-)